frontend/containers/layouts/navigation-menu-button/component.tsx

```diff
@@ -0,0 +1,124 @@
+import { FC, useState, useCallback } from 'react';
+
+import { Menu as MenuIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { useRouter } from 'next/router';
+
+import useMe from 'hooks/me';
+
+import Button from 'components/button';
+import Menu, { MenuItem, MenuSection } from 'components/menu';
+import { Paths } from 'enums';
+
+import { useCurrentProjectDeveloper } from 'services/project-developers/projectDevelopersService';
+
+import { NavigationMenuButtonProps } from './types';
+
+export const NavigationMenuButton: FC<NavigationMenuButtonProps> = ({
+  className,
+  theme = 'primary-white',
+}: NavigationMenuButtonProps) => {
+  const router = useRouter();
+  const intl = useIntl();
+  const { user } = useMe();
+  const { projectDeveloper } = useCurrentProjectDeveloper(user);
+  const investor = undefined; // Important!!! Include the current investor when available
+
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const onClickMenuItem = useCallback(
+    (key) => {
+      switch (key) {
+        default:
+          router.push(key);
+          break;
+      }
+    },
+    [router]
+  );
+
+  const account = projectDeveloper || investor;
+
+  return (
+    <div className={className}>
+      <div className="flex items-center">
+        <Menu
+          Trigger={
+            <Button
+              theme={theme}
+              size="small"
+              aria-expanded={menuOpen}
+              onClick={() => setMenuOpen(true)}
+              className="sm:py-2"
+            >
+              <span className="hidden sm:inline">
+                <FormattedMessage defaultMessage="Menu" id="tKMlOc" />
+              </span>
+              <span className="inline sm:hidden">
+                <MenuIcon height={20} />
+              </span>
+            </Button>
+          }
+          align="end"
+          onAction={onClickMenuItem}
+          hiddenSections={{ 'user-section': 'sm' }}
+        >
+          <MenuSection>
+            <MenuItem key="/discover">
```

Comment:
Maybe you could use the `Paths` enum values here? You might need to cast them.

Reply:
Thank you. Changed the hardcoded links to `Paths` in the navigation menus/bars touched.